lib/std/Thread.zig

```diff
@@ -75,7 +75,7 @@ pub fn spinLoopHint() callconv(.Inline) void {
         },
         .arm, .armeb, .thumb, .thumbeb => {
             // `yield` was introduced in v6k but are also available on v6m.
-            const can_yield = comptime std.Target.arm.featureSetHas(std.Target.current.cpu.features, .has_v6m);
+            const can_yield = comptime std.Target.arm.featureSetHasAny(std.Target.current.cpu.features, .{ .has_v6k, .has_v6m });
             if (can_yield) asm volatile ("yield" ::: "memory");
```

Comment:
Shouldn't this fallback to `asm volatile ("" ::: "memory")`?

Reply:
Good catch (even though I really doubt anyone is going to use that code path, heh).